src/Tools/Source/RunTests/Options.cs

```diff
@@ -182,38 +173,34 @@ internal class Options
                 return null;
             }
 
-            if (useProcDump && string.IsNullOrEmpty(procDumpFilePath))
+            if (retry && includeHtml)
             {
-                Console.WriteLine($"The option 'useprocdump' was specified but 'procdumppath' was not provided");
+                Console.WriteLine($"Cannot specify both --retry and --html");
                 return null;
             }
 
-            if (retry && includeHtml)
+            if (procDumpFilePath is { } && !collectDumps)
```

Comment:
We don't have `is not null` here?

Reply:
We do but `is not null` is an inferior check to `is {}` and `is object` hence I rarely use it.